### DISCUSSION TITLE
How to connect an old IDH node to a new Standalone management server?

### DISCUSSION THREAD

**Discussion by Johnny111125** [Security-Onion-Solutions/securityonion#13515] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

32 gb

### Storage for /

200 gb

### Storage for /nsm

150 gb

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi All,

To make a long story short, my management node failed when updating from 2.4.80 to 2.4.90. I was unable to fix it, so I decided to nuke it and do a fresh reinstall with 2.4.90. With that complete, I now need to reconnect my 2.4.80 IDH node to my 2.4.90 management node. The management server IP is the same as before, so I opened up the port on the management server and accept the IDH node.  I see the IDH node as an accepted member, however I do not see it listed in Grid. Are there any more steps I need to perform to get this working?

Thanks in advance! 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
If you did a complete reinstall on the Manager node, you're going to need to re-run setup on the minion nodes to make sure all of the certificates and such are correct.

First, remove it from the Grid Members. Then log into the IDH node via SSH, or use the console, and run "sudo ~/SecurityOnion/setup/so-setup iso" (assuming this node was installed from our ISO).

**Reply by Johnny111125** [reply]
Just did this, however I am having the same problem. Even wiped the IDH node and installed from ISO. I can see the IDH node in the Accepted Members list, but not in Grid. Are there old certs that need to be removed from the Manager node, maybe?

**Reply by InfosecGoon** [reply]
It might still be caching the old Salt key -- remove the node via the Grid Members screen, then log into the IDH node and run "sudo systemctl restart salt-minion" to reregister it.

**Reply by Johnny111125** [reply]
This worked. Thank you!
